install/lsstsw.rst

```diff
@@ -89,7 +89,15 @@ See :doc:`setup` for more information.
 =======================================
 
 Once the LSST Science Pipelines are installed, you can verify that it works by :doc:`running a demo project <demo>`.
-This demo processes a small amount of SDSS data.
+
+5. Upgrading your installation
+==============================
+
+You can upgrade an lsstsw installation in-place by running the following commands from within your :file:`lsstsw/` directory. Given that there may be ABI incompatibilities across conda environments, we suggest removing all of the sub-directories in your :file:`stack/VERSION/OS/` (where ``VERSION`` is the environment version and ``OS`` is the operating system name e.g. ``Linux64``) and `stack/ups_db/` paths.
+
+1. `git pull` to download the latest environment definition.
+2. `./bin/deploy` to install that new conda environment.
+3. `rebuild -u lsst_distrib` to download the latest repos definition file and rebuild the entire Science Pipelines codebase.
```

Comment:
I have encountered problems between steps 2 and 3 but all of which are solved by running 2 in one shell and then starting a new shell for 3.